test/parallel/test-child-process-pipe-dataflow.js

```diff
@@ -27,7 +27,7 @@ const MB = KB * KB;
   // So cut the buffer into lines at some points, forcing
   // data flow to be split in the stream.
   for (let i = 1; i < KB; i++)
-    buf.write(os.EOL, i * KB);
```

Comment:
What is the value of os.EOL? For windows I expecte it to be \r\n and wonder why cutting it down to just \n allows the test to pass.

Reply:
If you look at the error message here: https://ci.nodejs.org/job/node-test-binary-windows-js-suites/12290/RUN_SUBSET=3,nodes=win2012r2-COMPILED_BY-vs2019-x86/testReport/junit/(root)/test/parallel_test_child_process_pipe_dataflow/, you'll notice that the difference between the actual and the expected character count is 1023. That error can only be introduced if there is an additional (or missing) character per line, so I guess for some reason `\r` is not handled correctly.

Reply:
That would make sense if somewhere in the flow '\r\n' is getting converted to '\n'.

Reply:
I think it's the grep command not playing well with `CRLF`.

Reply:
I don't see any issues related to updating the Windows machines, but I'm not sure if they would update themselves @joaocgreis do you know.

Either way its worth getting the tests running again as long as this passes on all windows machines.

Reply:
I don't see any recent PRs that might have changed related behaviour in Node.js itself.